software/base/src/main/java/org/apache/brooklyn/entity/software/base/SoftwareProcessDriverLifecycleEffectorTasks.java

```diff
@@ -169,7 +173,7 @@ protected String startProcessesAtMachine(final Supplier<MachineLocation> machine
     }
 
     @Override
-    protected void postStartCustom() {
+    protected void postStartCustom(AtomicReference<ReleaseableLatch> startLatchRef) {
```

Comment:
I was wondering how to do that in a backwards way. Couldn't think of a good solution.
If we keep the old signature it won't get called so it's actually worse than getting a compile error.
Could do something with thread locals, calling the deprecated method and passing the argument. Do you think that's something worth doing?